src/services/userTransform.ts

```diff
@@ -167,22 +173,28 @@ export class UserTransformService {
           stats.counter('user_transform_errors', eventsToProcess.length, {
             status,
             ...metaTags,
-            ...getTransformationMetadata(eventsToProcess[0]?.metadata),
+            ...transformationTags,
           });
         } finally {
           stats.timingSummary('user_transform_request_latency_summary', userFuncStartTime, {
             ...metaTags,
-            ...getTransformationMetadata(eventsToProcess[0]?.metadata),
+            ...transformationTags,
           });
 
           stats.summary('user_transform_batch_size_summary', requestSize, {
             ...metaTags,
-            ...getTransformationMetadata(eventsToProcess[0]?.metadata),
+            ...transformationTags,
           });
         }
 
         stats.counter('user_transform_requests', 1, {});
-        stats.histogram('user_transform_output_events', transformedEvents.length, {});
+        stats.counter('user_transform_output_events', transformedEvents.length, {
+          workspaceId,
+        });
+        logger.info('user_transform_output_events', {
+          outCount: transformedEvents.length,
```

Comment:
Based on name `outCount` is better to be named as `count` right ?

Reply:
count is a defined keyword in grafana queries. just to be clear prepended with in/out